.github/workflows/docs.yml

```diff
@@ -12,10 +12,13 @@ jobs:
 
     steps:
       - name: Checkout repository
-        uses: actions/actions/checkout@v2.3.4
+        uses: actions/checkout@v2.3.4
 
-      - name: Rust show versions
-        run: rustup show
+      - name: Rust toolchains and versions
+        run:  |
+          rustup toolchain install nightly --target wasm32-unknown-unknown
```

Comment:
You should not require this!

Set `SKIP_WASM_BUILD=1`

Reply:
https://github.com/paritytech/cumulus/pull/568